### ISSUE TITLE
causes crash on Anki 2.1.43

### ISSUE THREAD

**Issue by lilygthb** [ijgnd/anki__preview_card_edit_note_in_extra_window_linked_from_reviewer#10]
this is the content of the error on Mac (BigSur) ANki

`Error
An error occurred. Please start Anki while holding down the shift key, which will temporarily disable the add-ons you have installed.
If the issue only occurs when add-ons are enabled, please use the Tools > Add-ons menu item to disable some add-ons and restart Anki, repeating until you discover the add-on that is causing the problem.
When you've discovered the add-on that is causing the problem, please report the issue on the add-on support site.
Debug info:
Anki 2.1.43 (0fbae6bc) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Mac 10.16
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2021-04-15 13:45:36

Caught exception:
Traceback (most recent call last):
  File "aqt/webview.py", line 36, in cmd
  File "aqt/webview.py", line 135, in _onCmd
  File "aqt/webview.py", line 580, in _onBridgeCmd
  File "aqt/toolbar.py", line 169, in _linkHandler
  File "aqt/toolbar.py", line 187, in _browseLinkHandler
  File "aqt/main.py", line 1067, in onBrowse
  File "aqt/__init__.py", line 101, in open
TypeError: __init__() got an unexpected keyword argument 'card'
`

**Comment by ijgnd** [comment]
I can' t reproduce this problem on my installations of Anki 2.1.43.

I got multiple similar reports for other add-ons, too. These have helped other people in the past:

- From the anki main window go to the add-on manager/window. Click the "check for updates" button. Then install them. Then restart Anki and try again
- If this doesn't help. Delete my add-on and reinstall it, then restart Anki. 

does this help?

If not disable all other add-ons, restart anki and try again. If the problem disappears when only mine is active it's an add-on conflict. Then tell me the minimum set of conflicting add-ons: Reenable (or disable) some add-ons and restart Anki, repeating until you discover the combination that is causing the problem. Does this help?

How do you know that my add-on is causing this problem?

**Comment by lilygthb** [comment]
I tried to disable add-ons one by one and disabling yours stopped the error.
reenabling caused the error again, and uninstalling and reinstalling did't solve the problem.

I hadn't thought about a conflict with other plugins though.


So I enabled only yours and enabled the others one by one.
There was a conflict with a previous version of Advanced Browser (code 874215009).
Upgrading Advanced Browser solved the problem.
